plugins/@grouparoo/customerio/src/lib/test.ts

```diff
@@ -8,6 +8,8 @@ export const test: TestPluginMethod = async ({ appOptions }) => {
     name: "grouparoo_test_connection",
   });
 
+  await client.getCampaigns();
```

Comment:
Is there any information we get from this response we can share with the user?  A message like `Connected.  Found 10 campaigns.` would be helpful